type_check.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
-mypy -p wwwapp
+DJANGO_SETTINGS_MODULE=wwwapp.settings mypy -p wwwapp
```

Comment:
Why is this required both here and in mypy.ini?

Reply:
It's an open issue with django-stubs, without this environment variable analysis fails because some django code is unable to find some parts of the configuration.